packages/vuetify/src/components/VTimeline/VTimelineDivider.tsx

```diff
@@ -39,14 +39,6 @@ export const VTimelineDivider = genericComponent()({
       backgroundColorStyles: lineColorStyles,
     } = useBackgroundColor(toRef(props, 'lineColor'))
 
-    provideDefaults({
-      VIcon: {
-        color: toRef(props, 'iconColor'),
-        icon: toRef(props, 'icon'),
-        size: toRef(props, 'size'),
```

Comment:
I don't know how much of a problem this actually is but provideDefaults allowed these to be overridden by the user with `VTimelineDivider: { VIcon: {...`